public/index.html

```diff
@@ -5,11 +5,9 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="logo192.png" />
+    <meta name="description" content="Web site created using create-react-app" />
```

Comment:
This description can be updated to include a brief description of this app.

Reply:
Thanks!